### PR TITLE
Extract text by quotes marks.

### PR DESCRIPTION
**Pull request by Victoria58**
## Extract text by quotes marks.

### CODE REVIEW

File: src/ProgramingSummer/StringExtractor/StringExtractor.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+public class StringExtractor {
+
+    public List<Integer> findQuotes(String inText) {
+
+        List<Integer> indexFromText = new ArrayList<>();
+
+        for (int i = 0; i < inText.length(); i++) {
+
+            if (inText.charAt(i) == '"') {
```
Comment (MaxAbrosimov) [inline src/ProgramingSummer/StringExtractor/StringExtractor.java]: change please `"` to` \"`

File: src/ProgramingSummer/StringExtractor/StringExtractor.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                indexFromText.add(i);
+            }
+        }
+        return indexFromText;
+    }
+
+    public List<String> extractString(String inText){
+
+        List<Integer> indexes = findQuotes(inText);
+        List<String> extractedStrings = new ArrayList<>();
```
Comment (MaxAbrosimov) [inline src/ProgramingSummer/StringExtractor/StringExtractor.java]: size of exctracted strings array will be sum of quotes / 2

File: src/ProgramingSummer/StringExtractor/StringExtractor.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
```
Comment (MaxAbrosimov) [inline src/ProgramingSummer/StringExtractor/StringExtractor.java]: add please comment what is going below

File: src/App.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+public class App {
+
+    public static void main(String[] args) {
+
+        String inText = "\"Xfd? gsdf\"-fd gdf gh-\"fgdf gdf g\".";
+        StringExtractor stringExtractor = new StringExtractor();
+        stringExtractor.extractString(inText);
+
+/*
```
Comment (MaxAbrosimov) [inline src/App.java]: u can remove it

**Review by MaxAbrosimov — CHANGES_REQUESTED** [review]

**Comment by MaxAbrosimov** [comment]
@Victoria58 please remove unneded blank lines

File: src/App.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        Pattern pattern = Pattern.compile("[\"]");
+        Matcher matcher = pattern.matcher(inText);
+
+        while (matcher.find()){
```
Comment (MaxAbrosimov) [inline src/App.java]: u can remove it

**Review by MaxAbrosimov — CHANGES_REQUESTED** [review]

File: src/App.java (hunk 5 above)
Comment (Victoria58) [reply src/App.java]: ok)

File: src/ProgramingSummer/StringExtractor/StringExtractor.java (hunk 1 above)
Comment (Victoria58) [reply src/ProgramingSummer/StringExtractor/StringExtractor.java]: silly

File: src/ProgramingSummer/StringExtractor/StringExtractor.java (hunk 3 above)
Comment (Victoria58) [reply src/ProgramingSummer/StringExtractor/StringExtractor.java]: ок

File: src/ProgramingSummer/StringExtractor/StringExtractor.java (hunk 2 above)
Comment (Victoria58) [reply src/ProgramingSummer/StringExtractor/StringExtractor.java]: ahha

**Pull request merged**